File: monitorus-res/uptime.php
<?php

# Данный скрипт выводит количество дней работы сервера без перезагрузки.

# Запустите данный скрипт в браузере, если ничего не выведется или появится ошибка, значит на Вашем сервере запрещена работа данного скрипта. В таком случае используйте Perl версию этого скрипта.
$res=`uptime`;

if (preg_match('/([0-9]+) day/i',$res,$matches)) {
	print $matches[1];
}
else {
	print '0';
}

?>

File: monitorus-res/uptime.pl
#!/usr/bin/perl

use strict;

# Данный скрипт выводит количество дней работы сервера без перезагрузки.

my $res=`uptime`;
print "Content-type: text/plain\n\n";
if ($res=~/([0-9]+) day/i) {
	print $1;
}
else {
	print '0';
}
